src/e2e-test/java/io/cdap/plugin/salesforcestreamingsource/stepsdesign/DesignTimeSteps.java

```diff
@@ -50,11 +73,59 @@ public void createNewLeadInSalesforce() throws UnsupportedEncodingException {
     lead.put("LastName", "LLname_" + uniqueId);
     lead.put("Company", uniqueId + ".com");
 
-    SalesforceClient.createLead(lead, "Lead");
+    SalesforceClient.createObject(lead, "Lead");
   }
 
   @Then("Enter unique Topic name as a Runtime argument value for key: {string}")
   public void fillUniqueTopicNameInRuntimeArguments(String runtimeArgumentKey) {
     SalesforcePropertiesPageActions.fillUniqueTopicNameInRuntimeArguments(runtimeArgumentKey);
   }
+
+  @Then("Update fields in salesforce")
+  public void updateFieldsInSalesforce() {
+    String uniqueRecordId = SalesforceClient.queryObjectId(SObjects.AUTOMATION_CUSTOM__C.value);
+    SalesforceClient.updateObject(uniqueRecordId, SObjects.AUTOMATION_CUSTOM__C.value);
+  }
+
+
+  @And("Create a new Custom Record in salesforce using REST API")
+  public void createANewCustomRecordInSalesforceUsingRESTAPI() throws UnsupportedEncodingException,
+    InterruptedException {
+    TimeUnit time = TimeUnit.SECONDS;
+    time.sleep(120);
+    JSONObject custom = new JSONObject(PluginPropertyUtils.pluginProp("testData"));
+    SalesforceClient.createObject(custom, SObjects.AUTOMATION_CUSTOM__C.value);
+  }
+  @Then("Validate records transferred from salesforce to big query is equal")
+  public void validateRecordsTransferredFromSalesforceToBigQueryIsEqual() throws IOException,
+    InterruptedException {
+    TimeUnit time = TimeUnit.SECONDS;
```

Comment:
Add a comment here that you are waiting for data to reflect in target table.

Reply:
done